src/Psalm/Issue/NonInvariantPropertyType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psalm\Issue;
+
+final class NonInvariantPropertyType extends CodeIssue
+{
+    public const ERROR_LEVEL = -1;
+    public const SHORTCODE = 235159;
```

Comment:
Thanks Bruce. Taking your suggestion:

```suggestion
    public const SHORTCODE = 265;
```

Reply:
I made https://github.com/vimeo/psalm/pull/5133 while I was there.